src/Environment.sol

```diff
@@ -15,9 +17,15 @@ import { PrizePool, ConstructorParams } from "v5-prize-pool/PrizePool.sol";
 import { Claimer } from "v5-vrgda-claimer/Claimer.sol";
 import { UD2x18, intoUD60x18 } from "prb-math/UD2x18.sol";
 import { SD1x18, unwrap, UNIT } from "prb-math/SD1x18.sol";
+import { SD59x18, convert } from "prb-math/SD59x18.sol";
 import { CommonBase } from "forge-std/Base.sol";
 import { StdCheats } from "forge-std/StdCheats.sol";
 
+import { ILiquidationSource as CgdaILiquidationSource } from "v5-cgda-liquidator/interfaces/ILiquidationSource.sol";
+import { LiquidationPair as CgdaLiquidationPair } from "v5-cgda-liquidator/LiquidationPair.sol";
```

Comment:
We should make an `ILiquidationPair` so the typing for `pair` on the Environment is a little clearer. I can handle that when adding third liquidator option.

Reply:
Good call!  Yes we need an interface for the Liquidation stuff.  Those castings are awful.

Ideally separate repo so it doesn't pull in all the extra junk